ekncontent/overrides/EosKnowledgeContent.js

```diff
@@ -54,14 +51,6 @@ function add_custom_model_constructors (model) {
 
 function _init() {
     let Eknc = this;
-
-    define_property(Eknc.ContentObjectModel, 'discovery-feed-content', {
```

Comment:
Do we want to return a `JSON.parse()` object, here, or an actual `JsonObject` and then use the json-glib API from JavaScript?